ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -924,7 +929,7 @@ def generate_bidirectional(self):
     def put_nowait(self, queue, data):
         try:
             queue.put_nowait(data)
-        except queue.Full:
+        except Queue.Full:
```

Comment:
This looks wrong as `Queue` is not same as `queue` arg that is passed to method.

Reply:
That line should have an exception type, not an instance. In my case, I got a Python error for `Full` being an unknown type.